pkgs/modules/python/pip.nix

```diff
@@ -18,6 +18,41 @@ let
     ];
   });
 
+  # This pip wrapper will detect if python is called from within
+  # the virtualenv: `.pythonlibs/bin`. If so, it will install a shell script
+  # called `pip` into the virtualenv so that if pip is called in the future
+  # python will think it's in the virtualenv.
+  pip-wrapper = pkgs.writeShellApplication {
+    name = "pip";
+    text = ''
+      python_location=$(which python)
+      venv_pip_location="''${REPL_HOME}/.pythonlibs/bin/pip"
+      venv_pip3_location="''${REPL_HOME}/.pythonlibs/bin/pip3"
+
+      function init_venv_pip {
+        cat <<EOF > "''${venv_pip_location}"
+      #! /bin/sh
+      export PIP_CONFIG_FILE=
```

Comment:
I tried this, I think we still need most of the properties in `pip.conf`, just not `user = false`

Reply:
These?
```
disable-pip-version-check = yes
break-system-packages = yes
```
Why?

Reply:
`break-system-packages = yes` is necessary if pip is running outside of the venv, but I guess it doesn't apply in this case anymore

Reply:
Resolving, feel free to proceed